### PR TITLE
Answer secondary category

### PR DESCRIPTION
**Pull request by natalimj**

### CODE REVIEW

File: src/api/AdminApi.ts [hunk 1]
```diff
@@ -43,20 +43,10 @@ const getAllQuestions = (accessToken: string) => {
...
-      theme: theme,
-      firstCategory: firstCategory,
-      secondCategory:secondCategory
-    },
-    headers: authHeader(accessToken)
-  }
-  )
+const addQuestion = (question: IQuestionData, accessToken: string) => {
+  console.log("in admin api")
+  console.log(question)
```
Comment (popaDaria) [inline src/api/AdminApi.ts:48]: can you remove the console.log messages?

File: src/components/EditQuestion.tsx [hunk 2]
```diff
@@ -108,7 +150,7 @@ const EditQuestion = ({ question, questions, setQuestions }: Props) => {
                             e2e-id={"question" + question.questionNumber + "EditText"}
                             defaultValue={question.questionText}
                             name="questionText"
-                            maxLength={150} />
+                            maxLength={500} />
```
Comment (popaDaria) [inline src/components/EditQuestion.tsx:153]: is this the backend limit too?

Comment (natalimj) [reply src/components/EditQuestion.tsx:153]: I didn't set a max limit for backend

File: src/util/Constants.tsx [hunk 3]
```diff
@@ -17,7 +17,9 @@ const Constants = {
 	categories : [
         { value: 'Select category', label: 'Select category', disabled: true },
         { value: 'Conservative', label: 'Conservative' },
-        { value: 'Progressive', label: 'Progressive' }
+        { value: 'Progressive', label: 'Progressive' },
+		{ value: 'Pragmatic', label: 'Pragmatic' },
```
Comment (popaDaria) [inline src/util/Constants.tsx:21]: weird alignment

**Pull request merged**